servers/src/main/java/tachyon/worker/block/BlockMasterSync.java

```diff
@@ -68,6 +70,8 @@
   private volatile boolean mRunning;
   /** The id of the worker */
   private long mWorkerId;
+  /** The thread pool to remove block*/
+  private final ExecutorService mFixedExecutor = Executors.newFixedThreadPool(10);
```

Comment:
Could you make the 10 a constant just to be more explicit? Also perhaps use `ThreadFactoryUtils` to create the `ThreadFactory`?